tools/update_lint_rules/lib/src/services/lint_rule_service.dart

```diff
@@ -39,6 +90,25 @@ class LintRuleService {
     );
   }
 
+  Future<Iterable<NotRecommendedRule>> getNotRecommendedRules() async {
+    // TODO: Reuse what has been obtained once.
```

Comment:
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey) 
I would like to address this in a separate pull request.